examples/model_selection/plot_det.py

```diff
@@ -83,22 +83,27 @@
 
 import matplotlib.pyplot as plt
 
+from sklearn.dummy import DummyClassifier
 from sklearn.metrics import DetCurveDisplay, RocCurveDisplay
 
 fig, [ax_roc, ax_det] = plt.subplots(1, 2, figsize=(11, 5))
 
-for name, clf in classifiers.items():
-    clf.fit(X_train, y_train)
-
-    RocCurveDisplay.from_estimator(clf, X_test, y_test, ax=ax_roc, name=name)
-    DetCurveDisplay.from_estimator(clf, X_test, y_test, ax=ax_det, name=name)
-
 ax_roc.set_title("Receiver Operating Characteristic (ROC) curves")
 ax_det.set_title("Detection Error Tradeoff (DET) curves")
 
 ax_roc.grid(linestyle="--")
 ax_det.grid(linestyle="--")
 
+for idx, (name, clf) in enumerate(classifiers.items()):
+    (color, linestyle) = ("black", "--") if idx == 2 else (None, None)
```

Comment:
Instead of using `id == 2` I would rather use a condition on `name` to make this more explicit.